jobs/openvpn/templates/etc/openvpn.conf.erb

```diff
@@ -26,6 +26,14 @@ server <%= p('openvpn.server') %>
 <% p('openvpn.push_routes').each do | route | %>
 push "route <%= route %>"
 <% end %>
+<% p('openvpn.push_dns').each do |dns| %>
+# ensure we can route to the DNS server over the connection
+push "route <%= dns %> 255.255.255.255"
+push "dhcp-option DNS <%= dns %>"
+<% end %>
+<% if_p('openvpn.push_dns_search_domains') do |domains| %>
+push "dhcp-option DOMAIN-SEARCH <%= domains.join(" ") %>"
```

Comment:
So, on my laptop (tunnelblick as the client), specifying `DOMAIN-SEARCH` once with multiple domains, only gives the first result. Specifying `DOMAIN-SEARCH` for each domain individually gives me multiple domains in my search path, and specifying multiple `DOMAIN` values gives me the last search path.

Looking through the openvpn code, the openvpn client looks like it only supports `DOMAIN`. If specified once, it sets `domain example.com` in resolv.conf, and if more than once, it switches to `search example.com other.com`.

Perhaps specify both DOMAIN and DOMAIN-SEARCH, once per domain for the best of all worlds?